private/lib/Controller/Category.php

```diff
@@ -185,6 +185,7 @@ public function setCategoryOrder(): void
             foreach ($sortOrders as $sortOrder => $categoryId) {
                 $categoryId = Sanitize::int($categoryId);
 
+                // sortOrder + 1  this way order starts from  1
```

Comment:
```suggestion
                // $sortOrder is incremented by one because we do not want sort order 0 to be used (reserved for uncategorized category)
```